janitor/src/init.lua

```diff
@@ -806,6 +794,7 @@ end
 export type Janitor = {
 	ClassName: "Janitor",
 	CurrentlyCleaning: boolean,
+	SuppressInstanceReDestroy: boolean,
```

Comment:
We need to have this type somewhere in our Janitor class so that people can modify it if they want.